packages/create-neutrino-project/bin/create-neutrino-project.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const yeoman = require('yeoman-environment');
+const Vorpal = require('vorpal');
```

Comment:
Since we did away with vorpal in favor of yargs in the Neutrino CLI, let's use that here as well.